lib/helpers/populate/setPopulatedVirtualValue.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+module.exports = function setPopulatedVirtualValue(populatedVirtuals, name, v, options) {
```

Comment:
maybe consider adding some jsdoc (at least types and descriptions of the parameters)